web_modules/common/_commonjsHelpers-b541825b.js

```diff
@@ -0,0 +1,11 @@
+var commonjsGlobal = typeof globalThis !== 'undefined' ? globalThis : typeof window !== 'undefined' ? window : typeof global !== 'undefined' ? global : typeof self !== 'undefined' ? self : {};
```

Comment:
what's with the name of this file? should this folder be gitignored?

Reply:
nm this might be related to branching off master not the snowpack branch. once that's merged I imagine this will go away

Reply:
I branched off of the snowpack branch. I think it would be pretty easy to cherry pick my changes into a fresh branch once that pr is merged.